client/src/elm/Pages/Prenatal/ProgressReport/View.elm

```diff
@@ -659,8 +659,43 @@ viewObstetricHistoryPane language currentDate measurements =
                     )
                 |> Maybe.withDefault []
 
+        obsetricHistoryStep2 =
```

Comment:
```suggestion
        obstetricHistoryHistoryStep2 =
```

For the sake of well written English names.
![image](https://github.com/user-attachments/assets/1e90c9af-4760-4d6b-a60e-95e4ae124dd0)

I guess it appears like this at a few places.